src/main/java/org/folio/rest/core/exceptions/ErrorCodes.java

```diff
@@ -114,7 +114,11 @@ public enum ErrorCodes {
   FUND_LOCATION_RESTRICTION_VIOLATION("fundLocationRestrictionViolation", "One of the funds is restricted to be used for one of the locations."),
   ENCUMBRANCES_FOR_RE_ENCUMBER_NOT_FOUND("encumbrancesForReEncumberNotFound", "The encumbrances were correctly created during the rollover or have already been updated."),
   CLAIMING_CONFIG_INVALID("claimingConfigInvalid", "Claiming interval should be set and greater than 0 if claiming is active"),
-  TEMPLATE_NAME_ALREADY_EXISTS("templateNameNotUnique", "Template name already exists");
+  TEMPLATE_NAME_ALREADY_EXISTS("templateNameNotUnique", "Template name already exists"),
+  INVALID_PHYSICAL_POL("physicalPOLShouldContainOnlyPhysicalElement", "Physical order format should contain only physical resource"),
+  INVALID_ELECTRONIC_POL("electronicPOLShouldContainOnlyElectronicElement", "Electronic order format should contain only electronic resource"),
+  INVALID_PEMIX_POL("peMixPOLShouldContainPhysicalAndElectronicElement", "PE mix order format should contain both of physical and electronic resource"),
+  INVALID_OTHER_POL("otherPOLShouldContainPhysicalElement", " Other order format should contain only physical resource");
```

Comment:
1 space to be deleted
" Other order format should contain only physical resource" -> "Other order format should contain only physical resource"
Also please check on our snapshot envs that we have the same UI validation, to be sure that this validation case is valid for Other POL: https://folio-snapshot.dev.folio.org/

Reply:
yes, it is checked :)

Reply:
the space was deleted